src/java.base/share/classes/java/time/LocalDate.java

```diff
@@ -1988,7 +1988,10 @@ public long toEpochSecond(LocalTime time, ZoneOffset offset) {
      * chronology is also considered, see {@link java.time.chrono.ChronoLocalDate#compareTo}.
      *
      * @param other  the other date to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value, that is less than zero if this is before {@code other},
```

Comment:
This text is incorrect. It does not match the description above re chronologies.